gensim/models/wrappers/dtmmodel.py

```diff
@@ -93,6 +93,9 @@ def __init__(
             lencorpus = sum(1 for _ in corpus)
         if lencorpus == 0:
             raise ValueError("cannot compute DTM over an empty corpus")
+        if model == "fixed" and any([i == 0 for i in [len(text) for text in corpus.get_texts()]]):
```

Comment:
Replace with `any(not text for text in corpus.get_texts())` (more Pythonic). This looks unnecessarily complicated.